magefiles/installation/install.go

```diff
@@ -0,0 +1,155 @@
+package installation
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/devfile/library/pkg/util"
+	"github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/config"
+	"github.com/go-git/go-git/v5/plumbing"
+	kubeCl "github.com/redhat-appstudio/e2e-tests/pkg/apis/kubernetes"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"k8s.io/klog"
+)
+
+const (
+	DEFAULT_INFRA_DEPLOYMENTS_BRANCH    = "main"
+	DEFAULT_INFRA_DEPLOYMENTS_GH_ORG    = "redhat-appstudio"
+	DEFAULT_LOCAL_FORK_NAME             = "qe"
+	DEFAULT_LOCAL_FORK_ORGANIZATION     = "redhat-appstudio-qe"
+	DEFAULT_E2E_APPLICATIONS_NAMEPSPACE = "appstudio-e2e-test"
+	DEFAULT_SHARED_SECRETS_NAMESPACE    = "build-templates"
+	DEFAULT_E2E_QUAY_ORG                = "redhat-appstudio-qe"
+)
+
+var (
+	previewInstallArgs = []string{"preview"}
```

Comment:
Please add `--keycloak` and `--tolchain` (they were added recently) https://github.com/redhat-appstudio/e2e-tests/blob/main/scripts/install-appstudio.sh#L39